index.html

```diff
@@ -24,14 +24,17 @@
             <div class="logo">
             <h1>My Logo</h1>
             </div>
-            <a href="#" class="bars-mob"></a>
-            <div class="nav-bar">
+            <a href="#" class="bars-mob" id="menu-bar"></a>
+            <nav >
+                <img class="bar-new" src="./images/close-menu.svg" alt="close-button" id="close-menu">
+            <div class="nav-bar bar-new">
                 <ul>
-                    <li><a href="#portfolio-1">Portfolio</a></li>
-                    <li><a href="#about-1">About</a></li>
-                    <li><a href="#contact-1">Contact</a></li>
+                    <li><a href="#portfolio-1" id="Portfolio">Portfolio</a></li>
+                    <li><a href="#about-1" id="About">About</a></li>
+                    <li><a href="#contact-1" id="Contact">Contact</a></li>
```

Comment:
- [ ] [OPTIONAL] It would be great if you don't make the navigation elements like the mobile menu in the desktop version 👍 

![image](https://user-images.githubusercontent.com/63915024/196479898-6dc14732-96b1-4bbd-b283-6fa1bb046a7c.png)